xyz-hub-service/src/main/java/com/here/xyz/hub/task/FeatureTaskHandler.java

```diff
@@ -39,6 +39,7 @@
 import static io.netty.handler.codec.http.HttpResponseStatus.TOO_MANY_REQUESTS;
 
 import com.fasterxml.jackson.core.JsonProcessingException;
+import com.google.protobuf.Value;
```

Comment:
As far as I can see, that import is not necessary.

Reply:
Fixed, thank you